src/style.css

```diff
@@ -2,0 +1,53 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+body {
+  margin: 0;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  margin: 20px 8px 0;
+}
+
+.search {
+  font-weight: 300;
+  background-image: url(images/Icon.png);
+  background-repeat: no-repeat;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  outline: none;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-family: Avenir, sans-serif;
```

Comment:
Here you correctly specified a fallback font, which is good. However, you should also include a generic font family at the end of the list. This acts as a final fallback in case none of the specified fonts are available. The generic families are `serif`, `sans-serif`, `cursive`, `fantasy`, and `monospace`.